tailwind.config.js

```diff
@@ -253,6 +253,7 @@ module.exports = {
       paragraph: '150%',
       code: '160%',
       huge: '170%',
+      18: '18px',
```

Comment:
I'd rather not have this here and stick with percentages as a general

Reply:
So inline it if you want to.

Reply:
ok